src/settings.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel
+import os
+import sys
+import logging
+
+
+class Settings(BaseModel):
+    """Class for keeping track of settings"""
+
+    guild_id: int
```

Comment:
might want to double check, but i was under the assumption that the guild id is a string

Reply:
The guild IDs I've seen when doing devwork have all been numeric, and the discord.py `Guild` class has id as an int, so I believe it's numeric